RecoVertex/GaussianSumVertexFit/test/GsfTest.cc

```diff
@@ -36,19 +89,14 @@ GsfTest::GsfTest(const edm::ParameterSet& iConfig) : theConfig(iConfig) {
 
 GsfTest::~GsfTest() { delete rootFile_; }
 
-void GsfTest::beginJob() {}
-
-void GsfTest::endJob() {}
-
 //
 // member functions
 //
 
 void GsfTest::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup) {
   if (not tree) {
-    edm::ESHandle<MagneticField> magField;
-    iSetup.get<IdealMagneticFieldRecord>().get(magField);
-    tree.reset(new SimpleVertexTree("VertexFitter", magField.product()));
+    const MagneticField* magField = &iSetup.getData(estoken_mf);
+    tree.reset(new SimpleVertexTree("VertexFitter", magField));
   }
 
   try {
```

Comment:
Could you please move to `cms::Exception`? Also, there are a few `cout` that can be changed to `edm::LogPrint`

Reply:
Done in the last push.